File: RS/src/main/java/com/yskc/rs/entity/init/InsuranceConfig.java
package com.yskc.rs.entity.init;

import java.io.Serializable;
import java.math.BigDecimal;
import java.util.Date;

import com.baomidou.mybatisplus.annotations.TableId;
import com.baomidou.mybatisplus.annotations.TableName;

/**
 * 
 * 
 * @author huronghua
 * @email 267267134@qq.com
 * @date 2019-07-30 14:56:07
 */
@TableName("insuranceConfig")
public class InsuranceConfig implements Serializable {
    private static final long serialVersionUID = 1L;

    /**
     *
     */
    @TableId
    private Integer id;
    /**
     *
     */
    private Integer companyId;
    /**
     * 养老
     */
    private BigDecimal endowment;
    /**
     * 医疗
     */
    private BigDecimal medical;
    /**
     * 失业
     */
    private BigDecimal unemployment;
    /**
     * 工伤
     */
    private BigDecimal injury;
    /**
     * 生育
     */
    private BigDecimal maternity;
    /**
     * 公积金
     */
    private BigDecimal house;

    /**
     *
     */
    private BigDecimal endowmentOfCom;
    /**
     *
     */
    private BigDecimal medicalOfCom;
    /**
     *
     */
    private BigDecimal unemploymentOfCom;
    /**
     *
     */
    private BigDecimal injuryOfCom;
    /**
     *
     */
    private BigDecimal maternityOfCom;
    /**
     *
     */
    private BigDecimal houseOfCom;

    private Integer creatorId;

    private Date createTime;

    private Integer updatorId;

    private Date lastUpdateTime;

    public BigDecimal getEndowmentOfCom() {
        return endowmentOfCom;
    }

    public void setEndowmentOfCom(BigDecimal endowmentOfCom) {
        this.endowmentOfCom = endowmentOfCom;
    }

    public BigDecimal getMedicalOfCom() {
        return medicalOfCom;
    }

    public void setMedicalOfCom(BigDecimal medicalOfCom) {
        this.medicalOfCom = medicalOfCom;
    }

    public BigDecimal getUnemploymentOfCom() {
        return unemploymentOfCom;
    }

    public void setUnemploymentOfCom(BigDecimal unemploymentOfCom) {
        this.unemploymentOfCom = unemploymentOfCom;
    }

    public BigDecimal getInjuryOfCom() {
        return injuryOfCom;
    }

    public void setInjuryOfCom(BigDecimal injuryOfCom) {
        this.injuryOfCom = injuryOfCom;
    }

    public BigDecimal getMaternityOfCom() {
        return maternityOfCom;
    }

    public void setMaternityOfCom(BigDecimal maternityOfCom) {
        this.maternityOfCom = maternityOfCom;
    }

    public BigDecimal getHouseOfCom() {
        return houseOfCom;
    }

    public void setHouseOfCom(BigDecimal houseOfCom) {
        this.houseOfCom = houseOfCom;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getCompanyId() {
        return companyId;
    }

    public void setCompanyId(Integer companyId) {
        this.companyId = companyId;
    }

    public BigDecimal getEndowment() {
        return endowment;
    }

    public void setEndowment(BigDecimal endowment) {
        this.endowment = endowment;
    }

    public BigDecimal getMedical() {
        return medical;
    }

    public void setMedical(BigDecimal medical) {
        this.medical = medical;
    }

    public BigDecimal getUnemployment() {
        return unemployment;
    }

    public void setUnemployment(BigDecimal unemployment) {
        this.unemployment = unemployment;
    }

    public BigDecimal getInjury() {
        return injury;
    }

    public void setInjury(BigDecimal injury) {
        this.injury = injury;
    }

    public BigDecimal getMaternity() {
        return maternity;
    }

    public void setMaternity(BigDecimal maternity) {
        this.maternity = maternity;
    }

    public BigDecimal getHouse() {
        return house;
    }

    public void setHouse(BigDecimal house) {
        this.house = house;
    }

    public Integer getCreatorId() {
        return creatorId;
    }

    public void setCreatorId(Integer creatorId) {
        this.creatorId = creatorId;
    }

    public Date getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }

    public Integer getUpdatorId() {
        return updatorId;
    }

    public void setUpdatorId(Integer updatorId) {
        this.updatorId = updatorId;
    }

    public Date getLastUpdateTime() {
        return lastUpdateTime;
    }

    public void setLastUpdateTime(Date lastUpdateTime) {
        this.lastUpdateTime = lastUpdateTime;
    }
}

File: YSDB/RSDB/Models/insuranceConfig.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace RSDB.Models
{
    /// <summary>
    /// 五险一金配置,按公司默认配置
    /// </summary>
    public class insuranceConfig : tablebase
    {
        [Key]
        public int id { get; set; }
        /// <summary>
        /// 公司Id
        /// </summary>
        public int companyId { get; set; }


        /// <summary>
        /// 养老
        /// </summary>
        [Column(TypeName = "decimal(6,4)")]
        public decimal endowment { get; set; }
        /// <summary>
        /// 医疗
        /// </summary>
        [Column(TypeName = "decimal(6,4)")]
        public decimal medical { get; set; }
        /// <summary>
        /// 失业
        /// </summary>
        [Column(TypeName = "decimal(6,4)")]
        public decimal unemployment { get; set; }
        /// <summary>
        /// 工伤
        /// </summary>
        [Column(TypeName = "decimal(6,4)")]
        public decimal injury { get; set; }
        /// <summary>
        /// 生育
        /// </summary>
        [Column(TypeName = "decimal(6,4)")]
        public decimal maternity { get; set; }
        /// <summary>
        /// 公积金
        /// </summary>
        [Column(TypeName = "decimal(6,4)")]
        public decimal house { get; set; }


        public int updatorId { get; set; }

        /// <summary>
        /// 养老,公司上交部分
        /// </summary>
        [Column(TypeName = "decimal(6,4)")]
        public decimal endowmentOfCom { get; set; }
        /// <summary>
        /// 医疗,公司上交部分
        /// </summary>
        [Column(TypeName = "decimal(6,4)")]
        public decimal medicalOfCom { get; set; }
        /// <summary>
        /// 失业,公司上交部分
        /// </summary>
        [Column(TypeName = "decimal(6,4)")]
        public decimal unemploymentOfCom { get; set; }
        /// <summary>
        /// 工伤,公司上交部分
        /// </summary>
        [Column(TypeName = "decimal(6,4)")]
        public decimal injuryOfCom { get; set; }
        /// <summary>
        /// 生育,公司上交部分
        /// </summary>
        [Column(TypeName = "decimal(6,4)")]
        public decimal maternityOfCom { get; set; }
        /// <summary>
        /// 公积金,公司上交部分
        /// </summary>
        [Column(TypeName = "decimal(6,4)")]
        public decimal houseOfCom { get; set; }

    }
}
